### PR TITLE
Add light theme to embed

### PR DESCRIPTION
**Pull request by asiia-trilitech**
## Proposed changes

[Task link](https://app.asana.com/0/1207646563234653/1207667420031331/f)

[Figma](https://www.figma.com/design/kUJJ7T75lP4GPNMg4GMVt1/Embed-%26-Web?node-id=783-50301&t=nhN1ZowBk3elDaW5-0)

## Types of changes

- [ ] Bugfix
- [x] New feature
- [ ] Refactor
- [ ] Breaking change
- [ ] UI fix

### Light Theme (default)

Screenshot 2024-07-22 at 15 26 18

### Dark Theme

Screenshot 2024-07-22 at 15 25 54

## Checklist

- [ ] Tests that prove my fix is effective or that my feature works have been added
- [ ] Documentation has been added (if appropriate)
- [ ] Screenshots are added (if any UI changes have been made)
- [ ] All TODOs have a corresponding task created (and the link is attached to it)

### CODE REVIEW

File: apps/embed-iframe/src/EmbeddedComponent.tsx [hunk 1]
```diff
@@ -102,6 +100,9 @@ export function EmbeddedComponent() {
         setUserData(null);
       }
     }
+    if (config.theme !== undefined) {
```
Comment (OKendigelyan) [inline apps/embed-iframe/src/EmbeddedComponent.tsx:103]: Could it be simplified to
```
if (config.theme) {
   ...
}
```

File: apps/embed-iframe/src/imported/style/theme.ts [hunk 2]
```diff
@@ -44,34 +44,109 @@ const sizes = {
...
+      },
+      defaultProps: {
+        variant: "primary",
+        size: "md",
+      },
+    },
+    Modal: {
+      baseStyle: (props: StyleFunctionProps) => ({
+        dialog: {
+          paddingTop: "36px",
```
Comment (OKendigelyan) [inline apps/embed-iframe/src/imported/style/theme.ts:87]: May be better to write as `padding: "36px 42px"`?

**Review by OKendigelyan — APPROVED** [review]
Approve with small comments 👍

File: apps/embed-iframe/src/imported/style/theme.ts (hunk 2 above)
Comment (asiia-trilitech) [reply apps/embed-iframe/src/imported/style/theme.ts]: Good catch - fixed 🙂

**Pull request merged**